### DISCUSSION TITLE
Is there any way to make background elements outside Dialogic interactable?

### DISCUSSION THREAD

**Discussion by ape1121** [dialogic-godot/dialogic#2132] [Q&A]
Even when Inputs subsystem does not handle the input, layout blocks mouse from passing.

Specifically I want to make a tutorial-like dialogue that requires user to interact with certain elements in game to progress the dialogue. What is the way to go with this?

**Comment by CakeVR** [comment]
Hello, what sort of background interaction have you considered?

There is a layer, part of the prebuilt styles, that's called `Input Catcher`.

Once input is not stopped (by UI elements) and reaches this layer, Dialogic will try to advance.
